src/app/models/pagecollection.js

```diff
@@ -0,0 +1,39 @@
+import BaseModel from '~/helpers/backbone/model';
+import BaseCollection from '~/helpers/backbone/collection';
+
+let urlOption = {
+        url: 'https://oscms-dev.openstax.org/api/v1/pages'
+    },
+    PageModel = BaseModel.extend(urlOption),
```

Comment:
You can use the new ES2015 class syntax rather than `extend`ing the model.  For example:
```javascript
@props({
    url: 'https://oscms-dev.openstax.org/api/v1/pages'
})
class PageModel extends BaseModel {
    
    initialize() {
        // initialize code
    }

    otherMethods() {
        // code
    }

}
```